ethereum-contracts/README.md

```diff
@@ -1,72 +1,61 @@
-# Testing contracts
+# Unidirectional Peggy Project Specification
 
-## Installing dependencies
+## Project Summary
+Unidirectional Peggy is the starting point for cross chain value transfers from Ethereum to Cosmos as part of the Ethereum Cosmos Bridge project. The smart contract system accepts incoming transfers of Ethereum and ERC20 tokens, locking them while the transaction is validated and equitable funds issued to the intended recipient on Cosmos.
```

Comment:
Can this part clarify that "Cosmos" is a specific bridge chain running Tendermint consensus and custom modules?

By itself, "Cosmos" refers to the network as a whole, not a particular blockchain.

Or is this intended to become an upgrade for the Cosmos Hub (which is a particular blockchain)?

Reply:
I believe that the custom modules will eventually be submitted to the Cosmos sdk as an upgrade for the Cosmos Hub and other blockchains Cosmos network. @Jazzy and @jackzampolin do you have any insight on this?